app/proxy/SWProxy.js

```diff
@@ -165,6 +165,11 @@ class SWProxy extends EventEmitter {
   getLogEntries() {
     return this.logEntries;
   }
+
+  clearLogs() {
+    this.log({ type: 'warning', source: 'proxy', message: 'clearLogEntries ran.' })
```

Comment:
Yeah, I had put that in to see when the function was being called, because when I was just clearing the entries, I opened the proxy and there was immediately nothing in the logs.